mesonbuild/mesondata.py

```diff
@@ -30,7 +30,126 @@
 # BEGIN Data section #
 ######################
 
-file_0_data_CMakeListsLLVM_txt = '''\
+file_0_data_preload_cmake = '''\
```

Comment:
Ouch! This is very messy to review. @mensinda this should be sorted...

Reply:
Ok, I'll relaunch the tool and rebase when this one is merged :)

Reply:
You can rebase now.